db-scheduler/src/test/java/com/github/kagkarlsson/scheduler/SchedulerClientTest.java

```diff
@@ -65,23 +66,27 @@ public void setUp() {
     }
 
     @Test
-    public void client_should_be_able_to_schedule_executions() {
+    public void client_should_be_able_to_schedule_executions() throws InterruptedException {
         SchedulerClient client = SchedulerClient.Builder.create(DB.getDataSource()).build();
         client.schedule(oneTimeTaskA.instance("1"), settableClock.now());
 
         scheduler.runAnyDueExecutions();
+        // Since execution is executed in an async way, we need to wait for a while to let the execution finish before asserting
+        Thread.sleep(1000);
```

Comment:
Sleeping in tests is an anti-pattern I am trying to avoid as much as possible :)